### ISSUE TITLE
Active app border color XFCE

### ISSUE THREAD

**Issue by Pyntux** [Fausto-Korpsvart/Everforest-GTK-Theme#16]
I do not know is Xfce supported, but nevertheless, with latest build (1.1) I do not have any active app border color (I was trying B-MB and B-LB variants), like border in menu (right click), or like border color on pictures of theme, here on git...

For now I am using some old variant (I think Everforest-Dark-B 0.9) and have borders...

![PXL_20240722_150233696.jpg](https://github.com/user-attachments/assets/a64a6739-4bff-41c8-9f14-6ff9d89be0de)

Is there a way to have those borders with lattest 1.1 theme on appd on xfce?

**Comment by Fausto-Korpsvart** [comment]
> Is there a way to have those borders with lattest 1.1 theme on appd on xfce?

Yes, the old version was a bit more customised, but in this new version the theme is a bit more generic, but with each update I will be adding more customisations.

**Comment by Pyntux** [comment]
Thank you!

**Comment by cowboycamilo** [comment]
Great theme...
How Can I chace the Folder color? I see lost of colors for the folders ...
I like the folder-white variant but can make it show..
![image](https://github.com/user-attachments/assets/b05fc653-646f-4231-986e-ec908f8498b3)

**Comment by Fausto-Korpsvart** [comment]
> Great theme... How Can I change the Folder color? I see lost of colors for the folders ... I like the folder-white variant but can make it show.. the script is only for theme... right?

I'm sorry I don't think I understand.
What loss of colour are you referring to?

I didn't create a white icon pack, per se, the white icon pack that is in this repo, refers to having support for mime type icons, the smaller icons, so that in panels, or in the sidebar of file explorers, or in context menus, so that they are visible, white icons in dark theme and black in light themes.

Yes, the script is only for the theme, I haven't actually created icon packs, those icons are generated and based on this theme [Suru Plus](https://github.com/gusbemacbe/suru-plus)

**Comment by cowboycamilo** [comment]
Ok, no worries..
I got It, i changed the symlinks on the "places" forde ot the dark theme...
I like how the "light/white" folders look more...
I kept the dark folders on the light theme..
Great work on the theme, many thanks!!!
[Dark](https://i.imgur.com/BQWPdGY.jpeg)
[Light](https://i.imgur.com/PAhDhR3.jpeg)
